modules/review_01/Guess the Number/Proj1StartVersion.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner; 
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Proj1StartVersion {
+    public static void main(String [] args){
+
+        int TargetNumber = ThreadLocalRandom.current().nextInt(-250, 250 + 1); 
+        Scanner GuessHolder = new Scanner( System.in );
+
+        System.out.println("give me your best guess: ");
+            int GuessReader = GuessHolder.nextShort();   
```

Comment:
yeah but I thought it was a good opportunity to use the Short data type since its range is from - 32,XXX to 32,XXX since our range is from -250 to 250, and you could bring up the point that it is more efficient to run the program this way.